app/assets/javascripts/app/components/comment_form/comment_form.js

```diff
@@ -13,7 +13,8 @@
       scope: {
         toggle: "&toggle",
         comment: "=comment",
-        discussion: "=discussion"
+        discussion: "=discussion",
+        isMobile: "=mobile"
```

Comment:
Why is this component receiving this property? Why can't you put `Device` on the dependencies?